src/sidebars/star.js

```diff
@@ -0,0 +1,49 @@
+﻿(function() {
+    'use strict';
+
+    angular.module('openfin.star', ['openfin.store'])
+        .controller('StarCtrl', ['$scope', 'storeService', function($scope, storeService) {
+            var self = this;
+
+            var favouriteColours = {
+                on: '#42D8BD',
+                off: '#1A1F26',
+                offhover: "#263337"
```

Comment:
Yes this should be improved I think. These colours are all already defined in our .less file anyway. Thankfully this refactor has brought all of these colours into one file. Should this be done as part of this PR or as a later tidy? I've already created an issue (#139) to tidy around the less file.